### ISSUE TITLE
 relation "messenger_messages" already exists 

### ISSUE THREAD

**Issue by Olivier4477** [jsor/doctrine-postgis#57]
hello, 

I apologize for creating an issue if it's a configuration problem on my part, but when I try to use the bundle I have this message:

```
In ExceptionConverter.php line 74:

  An exception occurred while executing a query: SQLSTATE[42P07]: Duplicate table: 7 ERROR:  relation "messenger_messages" already exists  


In Exception.php line 28:

  SQLSTATE[42P07]: Duplicate table: 7 ERROR:  relation "messenger_messages" already exists  


In Connection.php line 69:

  SQLSTATE[42P07]: Duplicate table: 7 ERROR:  relation "messenger_messages" already exists  
```
I tried with postgres 13 and 14..
I created my database then I activated postgis and added the field geography of my table.

```
     $this->addSql('CREATE TABLE messenger_messages (id BIGSERIAL NOT NULL, body TEXT NOT NULL, headers TEXT NOT NULL, queue_name VARCHAR(190) NOT NULL, created_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, available_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, delivered_at TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL, PRIMARY KEY(id))');
        $this->addSql('CREATE INDEX IDX_75EA56E0FB7336F0 ON messenger_messages (queue_name)');
        $this->addSql('CREATE INDEX IDX_75EA56E0E3BD61CE ON messenger_messages (available_at)');
        $this->addSql('CREATE INDEX IDX_75EA56E016BA31DB ON messenger_messages (delivered_at)');
        $this->addSql('CREATE TABLE messenger_messages (id BIGSERIAL NOT NULL, body TEXT NOT NULL, headers TEXT NOT NULL, queue_name VARCHAR(190) NOT NULL, created_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, available_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, delivered_at TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL, PRIMARY KEY(id))');
        $this->addSql('CREATE INDEX IDX_75EA56E0FB7336F0 ON messenger_messages (queue_name)');
        $this->addSql('CREATE INDEX IDX_75EA56E0E3BD61CE ON messenger_messages (available_at)');
        $this->addSql('CREATE INDEX IDX_75EA56E016BA31DB ON messenger_messages (delivered_at)');
        $this->addSql('CREATE OR REPLACE FUNCTION notify_messenger_messages() RETURNS TRIGGER AS $$
```

I deleted everything and I started again, I noticed when I add the postgis bundle, doctrine create me 2x the create table messenger... So I don't know if it's the postgis bundle that does this or symfony.

I deleted a creation and obviously it is compliant.

thank you so much

**Comment by davecosec** [comment]
i have this same issue,  @Olivier4477 

service:
    Jsor\Doctrine\PostGIS\Event\ORMSchemaEventSubscriber:
        tags: [{ name: doctrine.event_subscriber, connection: default }]
        
is causing this, i'm going to dig a bit deeper

**Comment by jsor** [comment]
This should be fixed with the latest [v2.2.0 release](https://github.com/jsor/doctrine-postgis/releases/tag/v2.2.0).

**Comment by davecosec** [comment]
Cheers mate, appreciated
